src/main/kotlin/step4/racingcar/racingcar/Car.kt

```diff
@@ -0,0 +1,25 @@
+package step4.racingcar.racingcar
+
+class Car(
+    val racerName: String
+) {
+    init {
+        if (racerName.length > RACER_NAME_LENGTH_LIMIT) {
+            throw IllegalArgumentException("입력된 선수의 이름이 5자 이상인 선수가 있습니다.")
+        }
+    }
+
+    private val _movements: MutableList<Boolean> = mutableListOf()
+
+    val movements
+        get() = _movements.toList()
+
+    fun moveOrStop(input: Int) {
+        _movements.add(input > GO_NUMBER)
+    }
```

Comment:
```suggestion
    fun move(input: Int) {
        _movements.add(input > GO_NUMBER)
    }
```

 자동차 안에서는 상황에 따라서 움직이고, 멈추겠지만, 자동차에게 움직여 달라는 요청을 보내는 객체에서는 그저, 움직여 달라는 메시지만 보내는 게 좋지 않을까요?

 그런 의미로 moveOrStop 보다, move라는 함수 이름이 더 적절하지 않을까요?